PixivUtil2.py

```diff
@@ -1489,6 +1489,10 @@ def main():
             __blacklistTitles = PixivTags.parseTagsList("blacklist_titles.txt")
             PixivHelper.print_and_log('info', 'Using Blacklist Titles: ' + str(len(__blacklistTitles)) + " items.")
 
+        if __config__.useBlacklistTitlesRegex:
+            global __blacklistTitlesRegex
+            __blacklistTitlesRegex = True
+
```

Comment:
I don't see the global `__blacklistTitlesRegex` being used anywhere? It is recommended to access the value from `__config__.useBlacklistTitlesRegex`, as the config might has been changed and reloaded.

I think these lines can be removed.